src/index.js

```diff
@@ -32,6 +34,23 @@ const editTask = (desc, index) => {
   addToLocalStorage();
 };
 
+const resetIndex = () => {
+  for (let i = 0; i < tasks.length; i += 1) {
+    tasks[i].index = tasks[i].index -= 1; //eslint-disable-line
```

Comment:
- [ ] Linters are helpful in ensuring that we have clean code and use the best practices so kindly remove this disable and please try to fix the error instead.